server/controllers/refer/index.ts

```diff
@@ -0,0 +1,14 @@
+/* istanbul ignore file */
+
+import ReferCaseListController from './caseListController'
+import type { Services } from '../../services'
+
+const controllers = (services: Services) => {
```

Comment:
I think we should keep the `refer/new/index` controllers, but I wonder if this top-level `refer/index` should be responsible for the `referNewControllers` as well, e.g. by importing them and spreading them into the return object? So then we just have one `referControllers` to import and spread at the top-level index? It's a lot of hierarchy, but it would match the file structure